packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -132,12 +132,6 @@ class RangeSlider extends StatefulWidget {
     this.activeColor,
     this.inactiveColor,
     this.semanticFormatterCallback,
-    @Deprecated(
-      'This flag has changed to true by default and no longer needed. '
-      'This feature was deprecated after v1.18.0.'
```

Comment:
We just announced 1.17 so we didn't get the release quite right here. I don't think it matters (anymore).

Reply:
When I run flutter --version it gives me 1.18.0. I wonder why.